migrating/checklists/ossm-migrating-kiali-differences-assembly.adoc

```diff
@@ -0,0 +1,62 @@
+// This module is used in the following assemblies:
+
+// * migrating/checklists/ossm-kiali-differences-assembly.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-kiali-differences-assembly_{context}]
+= Kiali differences for Service Mesh 3
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-kiali-differences-assembly
+
+toc::[]
+
+{kialiproduct} with {smproduct} introduces the following changes:
+
+* New topology graphs
+* Deprecated configuration settings
+* Renamed configuration settings
+
+== New topology graphs
+
+The **Traffic Page Graph** has been revamped and built using Patternfly topology, and a new topology view showcasing the mesh infrastructure has been added.
+
+== Deprecated configuration settings
+
+To control which namespaces are accessible or visible to users in {smproduct} 3, Kiali relies on `discoverySelectors`.
+
+By default, `deployment.cluster_wide_access=true` is enabled, granting Kiali cluster-wide access to all namespaces in the local cluster. If you are migrating a cluster-wide deployment with Kiali, you must remove the following deprecated and unavailable configuration settings from your Kiali custom resource (CR):
+
+* `spec.deployment.accessible_namespaces`
+* `api.namespaces.exclude`
+* `api.namespaces.include`
+* `api.namespaces.label_selector_exclude`
+* `api.namespaces.label_selector_include`
+
+If you are are using discovery selectors in Istio to restrict the namespaces that `Istiod` watches, then those must match the discovery selectors in your Kiali CR.
```

Comment:
Kiali DS should minimally be as restrictive as the Istio DS.  Although, Kiali could be even more restrictive, if desired.  I don't think you need to change what's here, it's clear and we do want users to do this, I'm just clarifying a bit further.